lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -250,16 +255,28 @@ class _CustomImageCropState extends State<CustomImageCrop>
     addTransition(CropImageData(x: event.delta.dx, y: event.delta.dy));
   }
 
-  Path _getPath(double cropWidth, double width, double height) {
+  Path _getPath(double cropWidth, double width, double height, bool clip) {
     switch (widget.shape) {
       case CustomCropShape.Circle:
+        if (clip) {
         return Path()
```

Comment:
formatting, please use `dart format .`

Reply:
I would also do the check to clip before the `switch`. This way if we add more shapes, you don't need to add the clip everywhere